mappings/net/minecraft/world/gen/ChunkRandom.mapping

```diff
@@ -2,32 +2,76 @@ CLASS net/minecraft/class_2919 net/minecraft/world/gen/ChunkRandom
+		COMMENT chunk. The coordinate pair provided is equivalent to (chunkX * 16, chunkZ * 16). The
+		COMMENT three values are mixed together through some layers of hashing to produce the
+		COMMENT population seed.
+		COMMENT
+		COMMENT This function has been proved to be reversible through some exploitation of the underlying
+		COMMENT nextLong() weaknesses. It is also important to remember that since setSeed()
+		COMMENT truncates the 16 upper bits of world seed, only the 48 lowest bits affect the population
+		COMMENT seed output.
 		ARG 1 worldSeed
-		ARG 3 x
-		ARG 4 z
-	METHOD method_12662 create (IIJJ)Ljava/util/Random;
-		ARG 0 x
-		ARG 1 z
+		ARG 3 posX
+		ARG 4 posZ
+	METHOD method_12662 getSlimeRandom (IIJJ)Ljava/util/Random;
+		ARG 0 chunkX
+		ARG 1 chunkZ
 		ARG 2 worldSeed
-		ARG 4 localSeed
-	METHOD method_12663 setStructureSeed (JII)J
-		ARG 1 worldSeed
-		ARG 3 x
-		ARG 4 z
-	METHOD method_12664 setFeatureSeed (JII)J
+		ARG 4 scrambler
+	METHOD method_12663 setCarverSeed (JII)J
+		COMMENT Seeds the randomizer to generate larger features such as caves, ravines, minehafts
+		COMMENT and strongholds.
+		COMMENT
+		COMMENT Similar to the population seed, only the 48 lowest bits of the world seed affect the
+		COMMENT output since it the upper 16 bits are truncated in the setSeed() call.
 		ARG 1 worldSeed
+		ARG 3 chunkX
+		ARG 4 chunkZ
+	METHOD method_12664 setDecoratorSeed (JII)J
+		COMMENT Seeds the randomizer to generate a given feature.
+		COMMENT
+		COMMENT @param populationSeed The population seed computed in setPopulationSeed().
+		COMMENT @param index The index of the feature in the feature list.
+		COMMENT @param step The generation step's ordinal for this feature.
+		COMMENT
+		COMMENT The salt, in the form of {index + 10000 * step} assures that each feature is seeded
+		COMMENT differently, making the decoration feel more random. Even though it does a good job
+		COMMENT at doing so, many entropy issues arise from the salt being so small and result in
+		COMMENT weird alignments between features that have an index close apart.
+		ARG 1 populationSeed
 		ARG 3 index
 		ARG 4 step
-	METHOD method_12665 setStructureSeed (JIII)J
+	METHOD method_12665 setRegionSeed (JIII)J
+		COMMENT Seeds the randomizer to determine the start position of structure features such as
+		COMMENT temples, monuments and buried treasures within a region.
+		COMMENT
+		COMMENT The region coordinates pair corresponds to the coordinates of the region the seeded
+		COMMENT chunk lies in. For example, a swamp hut region is 32 by 32 chunks meaning that all
+		COMMENT chunks that lie within that region get seeded the same way.
+		COMMENT
+		COMMENT Similarly, the upper 16 bits of world seed also do not affect the region seed because
+		COMMENT they get truncated in the setSeed() call.
 		ARG 1 worldSeed
-		ARG 3 x
-		ARG 4 z
-		ARG 5 seedModifier
-	METHOD next (I)I
+		ARG 3 regionX
+		ARG 4 regionZ
+		ARG 5 salt
+	METHOD next next (I)I
```

Comment:
You don't need to map unobfuscated methods.

Reply:
OOPS, I believe that was an accident.